mod.Lua

```diff
@@ -1,22 +1,20 @@
+local total_xp = 0 -- Create variables outside the hook, so we dont recreate them every hookcall
+local notification_text = ""
 
-local previous_total_xp = 0
-Hooks:PreHook(ExperienceManager, "mission_xp_award", "xp_notifier_before_xp_increase", function ()
-	previous_total_xp = get_current_total_xp()
-end)
```

Comment:
When trying to take XP bonuses into account, my first iteration was only using `Hooks:PostHook` plus a `total_xp` variable like here.

This led to some incorrect values when joining in the middle of the heist, though. This version with `PreHook` made it always consistent, although calling `get_current_total_xp()` twice isn't optimal.